FBTweak/FBTweakCollection.h

```diff
@@ -12,42 +12,48 @@
 @protocol FBTweak;
 
 /**
-  @abstract A named collection of tweaks.
+  @abstract A named collection of Tweaks.
  */
 @interface FBTweakCollection : NSObject <NSCoding>
 
+- (instancetype)init NS_UNAVAILABLE;
+
 /**
-  @abstract Creates a tweak collection.
-  @discussion This is the designated initializer.
+  @abstract Creates an empty Tweak collection.
  */
 - (instancetype)initWithName:(NSString *)name;
 
+/**
+ @abstract Creates a tweak category with an initial Tweak array.
+ */
+- (instancetype)initWithName:(NSString *)name tweaks:(NSArray<id<FBTweak>> *)tweaks NS_DESIGNATED_INITIALIZER;
+
 /**
   @abstract The name of the collection.
  */
 @property (nonatomic, copy, readonly) NSString *name;
 
 /**
-  @abstract The tweaks contained in this collection.
+  @abstract The Tweaks contained in this collection.
  */
-@property (nonatomic, copy, readonly) NSArray *tweaks;
+@property (nonatomic, copy, readonly) NSArray<id<FBTweak>> *tweaks;
 
 /**
-  @abstract Fetches a tweak by identifier.
-  @param identifier The tweak identifier to find.
-  @discussion Only search tweaks in this collection.
+  @abstract Fetches a Tweak by identifier.
+  @param Identifier The Tweak identifier to find.
+  @discussion Only search Tweaks in this collection.
```

Comment:
Add `returns \c nil if no Tweak with the given \c identifier was found.`?

Reply:
Fixed